src/Integration/Notifications/NotificationIndicatorViewModel.cs

```diff
@@ -30,38 +31,39 @@ namespace SonarLint.VisualStudio.Integration.Notifications
 {
     public class NotificationIndicatorViewModel : ViewModelBase, INotificationIndicatorViewModel
     {
+        private readonly ITimer autocloseTimer;
+        private readonly Action<Action> uiThreadInvoker;
+
         private string text;
         private bool hasUnreadEvents;
         private bool isVisible;
-        private bool isBalloonTooltipVisible;
+        private bool isToolTipVisible;
         private bool areNotificationsEnabled;
-        private readonly ITimer autocloseTimer;
-        private readonly Action<Action> uiThreadInvoker;
 
         public ObservableCollection<SonarQubeNotification> NotificationEvents { get; }
 
-        // For testing
         public NotificationIndicatorViewModel()
-            : this (ThreadHelper.Generic.Invoke)
+            : this(ThreadHelper.Generic.Invoke,
+                  new TimerWrapper { AutoReset = false, Interval = 3000 /* 3 sec */})
         {
         }
 
-        public NotificationIndicatorViewModel(Action<Action> uiThreadInvoker)
+        // For testing
+        public NotificationIndicatorViewModel(Action<Action> uiThreadInvoker, ITimer autocloseTimer)
```

Comment:
Feel free to ignore. I would make this one `internal` if only intended for testing.

Reply:
Hmm, I thought I did that already...